bluetooth-plane/CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# How to become a contributor and submit your own code
```

Comment:
We could just put this at the top level of the repo. Do we expect the rules to be different depending on the demo?